modin/experimental/core/execution/native/implementations/hdk_on_native/calcite_serializer.py

```diff
@@ -325,6 +338,22 @@ def serialize_literal(self, literal):
                 "type_scale": -2147483648,
                 "type_precision": 1,
             }
+        if isinstance(val, np.datetime64):
+            unit = np.datetime_data(val)[0]
+            precision = self._TIMESTAMP_PRECISION.get(unit, None)
+            if precision is not None:
+                if precision == 0 and unit != "s":
```

Comment:
Is this situation possible?

Reply:
@AndreyPavlenko?

Reply:
It was possible in the previous version, but now it's not. Thanks!